controllers/front/validation.php

```diff
@@ -48,7 +48,6 @@ public function postProcess()
         try {
             
             $validate->validate();
-            Configuration::updateValue('PAGSEGURO_CONFIGURATION_OK', true);
```

Comment:
Can you please bring back this configuration key ? It allows us to know when the module is properly configured.